servo/servo.py

```diff
@@ -233,19 +238,26 @@ async def dispatch_event(
             return await super().dispatch_event(*args, **kwargs)
 
     def __init__(
-        self, *args, connectors: list[servo.connector.BaseConnector], **kwargs
+        self,
+        *args,
+        connectors: list[servo.connector.BaseConnector] | None = None,
+        __connectors__: list[servo.connector.BaseConnector] | None = None,
+        **kwargs,
     ) -> None:  # noqa: D107
-        super().__init__(*args, connectors=[], **kwargs)
```

Comment:
The use of 'connectors' vs '__connectors__' is not easy to understand by looking at the code. If I got it right, the __connectors__ is exclusively for the 'events' mixin and is a copy of 'connectors', except it also includes 'self'.

A short comment explaining the use might be helpful here.

Reply:
I don't understand it 100% either but I think one is used for the event bus (includes servo) for dispatching events and the other is used by the servo (excludes servo self reference) for non-event collector handling

Reply:
The markdown let me down here - I meant `__connectors__` not __connectors__. Anyway, this is optional - but if you have an idea what to put in a docstring, it will be great.

Reply:
It should go in the events mixin but really there should be some logic to unify the variable so that its less confusing and the mixin is self-contained instead of having keyword arg details spilling out into other components.

https://github.com/opsani/servox/blob/f3ade34ce015fa64271f12d62518c0777d88ae81/servo/events.py#L646